SECP-service/src/main/resources/assets/app/views/password/forgot-password.html

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+    <div class="form-group">
+        <form autocomplete="on" novalidate name="emailForm">
+            <i class="fa fa-lock fa-4x"></i>
+            <h1>Forgot Password?</h1>
+            <p>You can reset your password here.</p>
```

Comment:
make this a span of the input.

Reply:
? I don't understand why. It's just a <p> tag thats there to add some info